### ISSUE TITLE
PHP version

### ISSUE THREAD

**Issue by TomaszKane** [MillionDollarScript/MillionDollarScript#46]
Hi!

> System Requirements
Million Dollar Script 2.1 and newer:
PHP 5.3+
by: https://milliondollarscript.com/category/requirements/system-requirements/

It's not true. In PHP 5.5 mysql extension is deprecated, and in PHP 7 is removed.
So this script probably works **fine** on PHP from 5.3 to 5.4.

**Comment by ty2u** [comment]
It should be using mysqli. I believe I tested it on PHP 7 and it worked ok.

**Comment by TomaszKane** [comment]
Oh shit! I clone repository and looking "2.1" branch - not found so thought that master is "the one".
Tomorrow I'll test "dev" branch on PHP 7 and after that this issue most probably can be closed :-)

**Comment by ty2u** [comment]
Sorry, I should have mentioned that but it slipped my mind since I haven't slept for far too long. The dev branch contains the changes I've made since I took over the development years ago. I want to work on it more but I just get so busy with other things and I'm working on rewriting it from scratch as a WordPress plugin. I'm not sure when that will be ready but it should be a lot nicer code than this.

**Comment by TomaszKane** [comment]
Nice.
BTW, I try to install "dev" branch on PHP 7 and MySQL 5.5, got errors like:
`Error: Invalid default value for 'ad_date'` : `ad_date datetime default CURRENT_TIMESTAMP,`
MySQL doc says that CURRENT_TIMESTAMP works with datetime column from 5.6 version.
http://dev.mysql.com/doc/refman/5.6/en/timestamp-initialization.html
So System Requirements should says MySQL 5.6+ ?

**Comment by ty2u** [comment]
That was due to the db not working for someone so I changed the db install queries a bit and it broke support for older MySQL versions in the process. I intended to fix it somehow but then there was a huge task to support newer MySQL versions at one point. I spent a number of hours working on it but I didn't feel like it was even close to finished and I really didn't want to do all that work on the old version when I really would rather work on the new WP plugin. It's such a mess, I know. I apologize.
